core/services/AuthService.js

```diff
@@ -18,10 +18,17 @@ module.exports = {
     throw new Error("No such operator or wrong password provided");
   },
 
-  generateToken: operator => jwt.sign(_.pick(operator, ...AuthService.OPERATOR_ATTRIBUTES_FOR_JWT), config.get("jwt.secret"), {expiresIn: config.get("jwt.lifetime")}),
+  generateToken: operator => jwt.sign(
+    _.pick(operator, ...AuthService.OPERATOR_ATTRIBUTES_FOR_JWT),
+    config.get("jwt.secret"),
+    {expiresIn: config.get("jwt.lifetime")}
+  ),
 
   checkAndDecodeToken: (token) => {
     const result = jwt.verify(token, config.get("jwt.secret"));
     return _.omit(result, "iat");
-  }
+  },
+
+  checkRight: (operator, modelKey, right) =>
+    !!(operator.isAdministrator || (operator.rights[modelKey.toLowerCase()] && operator.rights[modelKey.toLowerCase()].includes(right.toLowerCase())))
```

Comment:
just add `operator.rights && ` before the `operator.rights[modelKey.toLowerCase()] &&` block for the case when `model.rights` is empty.